docs/sources/administration/provisioning.md

```diff
@@ -141,13 +141,15 @@ Please refer to each datasource documentation for specific provisioning examples
 
 Since not all datasources have the same configuration settings we only have the most common ones as fields. The rest should be stored as a json blob in the `jsonData` field. Here are the most common settings that the core datasources use.
 
+> **Note:** Datasources denoted _HTTP\*_ below means any datasource communicating using the HTTP protocol, e.g. all core datasources except MySQL, PostgreSQL and MSSQL.
```

Comment:
Think parts of this could be improved slightly. WDYT?

```suggestion
> **Note:** Datasources tagged with _HTTP\*_ below denotes any data source which communicates using the HTTP protocol, e.g. all core data source plugins except MySQL, PostgreSQL and MSSQL.
```